patches/minecraft/net/minecraft/entity/player/ServerPlayerEntity.java.patch

```diff
@@ -19,7 +19,7 @@
        ServerWorld serverworld = this.func_71121_q();
        RegistryKey<World> registrykey = serverworld.func_234923_W_();
 -      if (registrykey == World.field_234920_i_ && p_241206_1_.func_234923_W_() == World.field_234918_g_) {
-+      if (registrykey == World.field_234920_i_ && p_241206_1_.func_234923_W_() == World.field_234918_g_ && teleporter.isVanilla()) { //Forge: Fix non-vanilla teleporters triggering end credits
++      if (registrykey == World.field_234920_i_ && p_241206_1_.func_234923_W_() == World.field_234918_g_) { //Forge: Fix non-vanilla teleporters triggering end credits
```

Comment:
Probably don't need the comment here anymore, since you are no longer changing code on this line

Reply:
Yeah just saw that. Still going through things

Reply:
Are you sure this line doesn't need to be modified still? Try using `forge setdimension` to go from the end to the overworld. That is why I initially had to make a PR to add the isVanilla check to this line (Though I believe it might have been an inlined class check back then)

Reply:
Are you sure this line doesn't need to be modified still? Try using forge setdimension to go from the end to the overworld. That is why I initially had to make a PR to add the isVanilla check to this line (Though I believe it might have been an inlined class check back then)

Reply:
Keeping the isVanilla check here so that it doesn't show end credits